SFDigitalServices/sf-dahlia-web
# DAH-1832 feat: filter out Veterans Preferences from ListingDetailsPre…

Jira: https://sfgovdt.jira.com/browse/DAH-1832
# Changes
- Filter our preferences to exclude any preferences with a `preferenceName` that includes "Veterans"

# Review Instructions
- Go to a listing with Veteran Preferences applied: /listings/a0W8H0000014Yo4UAE?react=true
- Scroll down to Lottery Preferences and ensure that no Veteran preference types appear
- Click View Lottery Results button and you should see Veteran Preferences here

Demo: https://www.loom.com/share/f92306c70d064134a27478a8050b54ba?sid=0eadfb80-115f-474c-9565-27b14c5b27ef
